src/handlers/actor_handler.go

```diff
@@ -0,0 +1,35 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/controllers"
+	"github.com/ELITE-Kinoticketsystem/Backend-KTS/src/utils"
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+)
+
+func GetActorByIdHandler(actorController controllers.ActorControllerI) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		actorId := uuid.MustParse(c.Param("id"))
```

Comment:
`MustParse()` panics on error, use `Parse()` and handle the error

should be fine in tests though